SmartBook/SmartBook/View/Smart Services/Servie Provider Profile/ServiceProviderProfileViewModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ServiceProviderProfileViewModel.swift
+//  SmartBook
+//
+//  Created by Md. Kamrul Hasan on 29/9/25.
+//
+
+import UIKit
+
+final class ServiceProviderProfileViewModel {
+    
+    private var serviceProvider: ServiceProvider
+    
+    init(serviceProvider: ServiceProvider) {
+        self.serviceProvider = serviceProvider
+    }
+    
+    var nameText: String { serviceProvider.name ?? "Unknown" }
+    var experienceText: String { "\(serviceProvider.experience ?? 0) yrs experience" }
```

Comment:
use new line for these computed vars -
```suggestion
    var experienceText: String { 
        "\(serviceProvider.experience ?? 0) yrs experience"
    }
```